views/dapp/views/mail-market-pool/components/mail-market-pool-risk/index.tsx

```diff
@@ -6,7 +6,11 @@ import { ProgressSVG } from '@/svg';
 
 import { MAILMarketPoolRiskProps } from '../../mail-market-pool.types';
 
-const MAILMarketPoolRisk: FC<MAILMarketPoolRiskProps> = ({ loading, risk }) => (
+const MAILMarketPoolRisk: FC<MAILMarketPoolRiskProps> = ({
+  loading,
+  risk,
+  text,
```

Comment:
dont change the interface of the components. What if we add more text one day.

call the hook here